TicTacToe/Validation.cs

```diff
@@ -3,16 +3,23 @@
 namespace TicTacToe
 {
     public class Validation
-    {
+    {

```

Comment:
what happens when a player inserts special characters or numbers? we should consider adding validation for that as well